.github/workflows/e2e-test.yml

```diff
@@ -121,131 +62,11 @@ jobs:
-                extra_hosts:
-                  - "$NEXUS_HOSTNAME:$IP_DOCKER_COMPOSE_BRIDGE"
-
-      #[distutils]
-      #index-servers =
-      #    nexus
-      #
-      #[nexus]
-      #repository =  http://$NEXUS_HOSTNAME:$PORT_NEXUS/repository/$REPO_NAME_PYPI/
-      #username = $NEXUS_USER_NAME
-      #password = $NEXUS_USER_PWD
-      - name: modify pypi repository
-        uses: "finnp/create-file-action@master"
+      - name: run e2e
+        working-directory: ./scripts/e2e_test
         env:
-          FILE_NAME: "~/.pypirc"
-          FILE_CONTENT: |
-            [distutils]
-            index-servers =
-                nexus
-            
-            [nexus]
-            repository =  http://$NEXUS_HOSTNAME:$PORT_NEXUS/repository/$REPO_NAME_PYPI/
-            username = $NEXUS_USER_NAME
-            password = $NEXUS_USER_PWD
-
-      - name: wait nexsu service up
-        uses: ifaxity/wait-on-action
-        with:
-          resource: http-get://$NEXUS_HOSTNAME:PORT_NEXUS
-
-      - name: create docker/pypi repository in nexus
-        run: |
-            curl -u $NEXUS_USER_NAME:$NEXUS_USER_PWD -X 'POST' 'http://$NEXUS_HOSTNAME:8081/service/rest/v1/repositories/docker/hosted' -H 'accept: application/json' -H 'Content-Type: application/json'  -d '{"name":"$REPO_NAME_DOCKER","online":true,"storage":{"blobStoreName":"default","strictContentTypeValidation":true,"writePolicy":"allow_once"},"component":{"proprietaryComponents":true},"docker":{"v1Enabled":false,"forceBasicAuth":false,"httpPort":$PORT_NEXUS_DOCKER}}'
-            curl -u $NEXUS_USER_NAME:$NEXUS_USER_PWD -X 'PUT' 'http://$NEXUS_HOSTNAME:8081/service/rest/v1/security/realms/active'  -H 'accept: application/json' -H 'Content-Type: application/json' -d '["DockerToken","NexusAuthenticatingRealm", "NexusAuthorizingRealm"]'
-            curl -u $NEXUS_USER_NAME:$NEXUS_USER_PWD -X 'POST' 'http://$NEXUS_HOSTNAME:8081/service/rest/v1/repositories/pypi/hosted' -H 'accept: application/json' -H 'Content-Type: application/json' -d '{"name":"$REPO_NAME_PYPI","online":true,"storage":{"blobStoreName":"default","strictContentTypeValidation":true,"writePolicy":"allow_once"},"component":{"proprietaryComponents":true}}'
-
-
-      - name: docker push to nexus
-        run: |
-            docker login http://$NEXUS_HOSTNAME:$PORT_NEXUS_DOCKER -u $NEXUS_USER_NAME -p $NEXUS_USER_PWD
-            docker push $NEXUS_HOSTNAME:$PORT_NEXUS_DOCKER/server
-            docker push $NEXUS_HOSTNAME:$PORT_NEXUS_DOCKER/starwhale
-            
-      - name: upload pypi to nexus
-        run:
-            twine upload --repository nexus dist/*
-
-      - name: start docker compose
-        working-directory: ./docker/compose
-        run: docker-compose up
-
-      - name: wait docker service up
-        uses: ifaxity/wait-on-action
-        with:
-          resource: http-get://localhost:8082
-
-      - name: run swcli stand alone e2e
-
-      - name: run server e2e
-
+          GITHUB_ACTION: 1
```

Comment:
Remove `GITHUB_ACTION` env?   [Github action](https://docs.github.com/en/actions/learn-github-actions/environment-variables) will inject the env automatically.

Reply:
done